doc/TUTORIALS_CN.md

```diff
@@ -869,6 +871,281 @@ Rosetta 完整代码参考 [rtt-ds-lr.py](../example/tutorials/code/rtt-ds-lr.py
+   batch_size=batch_size, 
+   model = model, 
+   optimizer = optimizer, 
+   loss = loss, 
+   accuracy_function = accuracy_function, 
+   X_test = X_test, 
+   Y_test = Y_test
+   )
+```
+
+完整代码参考[tf-mlp_mnist.py](../example/tutorials/code/tf-mlp_mnist.py)
+
+执行以下命令即可运行
+
+```python
+python ./tf-mlp_mnist.py
+```
+
+输出如下：
+
+```python
+epoch: 0000 loss = 17.504000
+epoch: 0001 loss = 6.774922
+epoch: 0002 loss = 4.993065
+epoch: 0003 loss = 4.047511
+epoch: 0004 loss = 3.440471
+epoch: 0005 loss = 3.006049
+  ...
+epoch: 0027 loss = 0.874316
+epoch: 0028 loss = 0.847307
+epoch: 0029 loss = 0.821776
+accuracy=0.91100000
+```
+
+#### Rosetta版
+
+- 导包并激活协议
+
+```python
+import os
+import tensorflow as tf
+import latticex.rosetta as rtt
+import csv
+import numpy as np
+
+rtt.set_backend_loglevel(1)
+np.set_printoptions(suppress=True)
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '2'
+np.random.seed(0)
+#激活协议
+rtt.activate("SecureNN")
+mpc_player_id = rtt.py_protocol_handler.get_party_id()
+```
+
+- 加载数据集
+
+这一部分数据集使用的是加载超大数据集的方法。具体介绍可参考[支持超大数据集](#支持超大数据集)。
+
+```python
+# load data
+file_x = '../dsets/P' + str(mpc_player_id) + "/mnist_train_x.csv"
+file_y = '../dsets/P' + str(mpc_player_id) + "/mnist_train_y.csv"
+X_train_0 = rtt.PrivateTextLineDataset(file_x, data_owner=0)
+X_train_1 = rtt.PrivateTextLineDataset(file_x, data_owner=1)
+Y_train = rtt.PrivateTextLineDataset(file_y, data_owner=1)
+
+# dataset decode
+def decode_p0(line):
+    fields = tf.string_split([line], ',').values
+    fields = rtt.PrivateInput(fields, data_owner=0)
+    return fields
+def decode_p1(line):
+    fields = tf.string_split([line], ',').values
+    fields = rtt.PrivateInput(fields, data_owner=1)
+    return fields
+  
+# dataset pipeline
+X_train_0 = X_train_0.map(decode_p0).cache(f"{cache_dir}/cache_p0_x0").batch(BATCH_SIZE).repeat()
+X_train_1 = X_train_1.map(decode_p1).cache(f"{cache_dir}/cache_p1_x1").batch(BATCH_SIZE).repeat()
+Y_train = Y_train.map(decode_p1).cache(f"{cache_dir}/cache_p1_y").batch(BATCH_SIZE).repeat()
+```
+
+- 这个例子中可以直接将训练的模型进行保存
+
+```python
+def tensorflow_classification(n_epochs, n_batches,
+                              batch_size,
+                              model, optimizer, loss
+                              ):
+    with tf.Session() as tfs:
+        tfs.run(tf.global_variables_initializer())
+        for epoch in range(n_epochs):
+            epoch_loss = 0.0
+            for i in range(n_batches):
+                X_batch = X_train[(i * batch_size):(i + 1) * batch_size]
+                Y_batch = Y_train[(i * batch_size):(i + 1) * batch_size]
+                feed_dict = {x: X_batch, y: Y_batch}
+                tfs.run([optimizer, loss], feed_dict)
+        saver.save(tfs, './log/ckpt'+str(mpc_player_id)+'/model')
+```
+
+其余部分与`tensorflow`版的基本相同，完整代码请参考[rtt-mlp_mnist.py](../example/tutorials/code/rtt-mlp_mnist.py)
+
+执行
+
+```python
+./tutorials.sh rtt mlp_mnist
+```
+
+明文模型会保存在`log/`文件夹下，可以继续执行
+
+```python
+python tf-test_mlp.py
```

Comment:
The name `tf-test_mlp.py`  in this doc is different with the script name `tf-test_mlp_acc.py`. It this a typo?